### PR TITLE
Added Event for delegation Accepted

### LINKED ISSUE #264: Emit an event on calling setDelegationAcceptance
As per the current contracts, no event is being emitted from contracts on `setDelegationAcceptance`.

It would be good to have an event that gets emitted while calling `setDelegationAcceptance`. This would help while getting stakerInfo.

### PR DESCRIPTION
**Pull request by adi44**
Fixes #264

### CODE REVIEW

File: contracts/Core/StakeManager.sol [hunk 1]
```diff
@@ -328,8 +330,10 @@ contract StakeManager is Initializable, ACL, StakeStorage, Pause {
     /// @notice Used by staker to set delegation acceptance, its set as False by default
     function setDelegationAcceptance(bool status) external {
         uint256 stakerId = stakerIds[msg.sender];
+        address stakerAddress = stakers[stakerId]._address;
         require(stakerId != 0, "staker id = 0");
         stakers[stakerId].acceptDelegation = status;
+        emit DelegationEnabled(stakerId, stakerAddress, status);
```
Comment (abhishekvispute) [inline contracts/Core/StakeManager.sol:336]: we can directly send msg.sender as address here @adi44

File: contracts/Core/StakeManager.sol [hunk 2]
```diff
@@ -328,8 +330,10 @@ contract StakeManager is Initializable, ACL, StakeStorage, Pause {
     /// @notice Used by staker to set delegation acceptance, its set as False by default
     function setDelegationAcceptance(bool status) external {
         uint256 stakerId = stakerIds[msg.sender];
+        address stakerAddress = stakers[stakerId]._address;
```
Comment (abhishekvispute) [inline contracts/Core/StakeManager.sol:333]: ..

Comment (adi44) [reply contracts/Core/StakeManager.sol]: yeah. we can pass msg.sender

File: contracts/Core/StakeManager.sol [hunk 3]
```diff
@@ -134,6 +134,8 @@ contract StakeManager is Initializable, ACL, StakeStorage, Pause {
 
     event Delegated(uint256 epoch, uint256 indexed stakerId, address delegator, uint256 previousStake, uint256 newStake, uint256 timestamp);
 
+    event DelegationEnabled(uint256 indexed stakerId, address staker, bool delegationEnabled);
```
Comment (abhishekvispute) [inline contracts/Core/StakeManager.sol:137]: Given event is also going to be fired when delegation is disabled 
so maybe "DelegationEnabled" is not correct as the name 
can we have DelegationAcceptanceChanged ?

**Comment by adi44** [comment]
Yeah. Even I was thinking of something like this

**Review by abhishekvispute — APPROVED** [review]

**Review by hrishikeshio — APPROVED** [review]

**Pull request merged**